.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,22 @@
+name: Enhancement
+description: Suggest a new feature or improvement for the website.
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to create your feature request!
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe what feature(s) you want.
+      description: "Describe the enhancement in as much detail as possible. What does it do? What problem does it solve? How might it work?"
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional-info
+    attributes:
+      label: Additional Information
+      description: "Add any other information about your feature request here."
```

Comment:
My main reason for not wanting this field is that if no-one uses it, we'll have a bunch of issues that have an "Additional Information" field with nothing underneath it.
```suggestion
```